change/@ni-nimble-components-cd11a25d-60af-4885-bea7-45694aae848f.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
Can you bump the guid in preview.js to rebuild all stories? I was expecting a snapshot update for the tokens story as well, so let's recalculate all just in case: https://github.com/ni/nimble/blob/9a302d397a783375deced506050a0a3f95b4a301/packages/storybook/.storybook/preview.js#L108

Reply:
Updated with new GUID